app/marsRoverPath/index.js

```diff
@@ -0,0 +1,45 @@
+var dragItem = document.querySelector('#rover-sprite');
```

Comment:
Could make sense to group this declaration with its usages a few lines down

Reply:
Done. :)